CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.1.0]
+### Added
+- Allow prerelease versions in release headers ([#130](https://github.com/MetaMask/auto-changelog/pull/130))
+
+### Changed
+- Check for `code` property when interpreting filesystem errors ([#124](https://github.com/MetaMask/auto-changelog/pull/124))
```

Comment:
Not sure if this is worth noting or not.

Reply:
Hmm. I think the only functional difference is that if a primitive type is thrown, we now re-throw it rather than crashing again when trying to check the `code` property`. Otherwise it should be functionally identical. We were checking the `code` property before, just not verifying that the error was an error before checking.

I'd probably remove it or re-word it to reflect the functional impact it has, but it's not a big deal.

Reply:
Ah, that's a good point. I was leaning toward removing this; it's kind of hard to explain, and I don't think people are likely to run into this.